src/SFA.DAS.AssessorService.Web/Controllers/ManageUsers/InviteUserController.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.AssessorService.Web.Infrastructure;
+
+namespace SFA.DAS.AssessorService.Web.Controllers.ManageUsers
+{
+    [Authorize]
+    [CheckSession]
+    public class InviteUserController : Controller
+    {
+        public InviteUserController()
+        {
```

Comment:
I assume there will be some sort of implementation in here later 😉

Reply:
Yes, as I mentioned, the implementation is in a different ticket.  This is just for the main page.